packages/core/EventManager.ts

```diff
@@ -377,36 +379,49 @@ export default class EventManager {
     event: CalendarEvent,
     booking: PartialBooking
   ): Promise<Array<EventResult<NewCalendarEventType>>> {
-    return async.mapLimit(this.calendarCredentials, 5, async (credential: Credential) => {
```

Comment:
Previous behaviour was to loop through every calendar credential. This was causing the errors and was adding unnecessary work. Should increase performance